README.md

```diff
@@ -1,2 +1,32 @@
+<p align="center">
+  <a href="#">
+    <img width="150" src="logo.png">
+  </a>
+</p>
+
 # eslint-config-extensionengine
 Extension Engine's eslint config
+
+## Usage
+
+### Install
+
+Use `install-peerdeps`
+
+```
+npx install-peerdeps --dev eslint-config-extensionengine
```

Comment:
I was under impression we'll publish it under `@extensionengine` scope: `@extensionengine/eslint-config`?